mappings/net/minecraft/world/World.mapping

```diff
@@ -20,12 +61,94 @@ CLASS net/minecraft/class_18 net/minecraft/world/World
 		ARG 3 dimension
 		ARG 4 seed
 	METHOD method_149 getClosestPlayer (DDDD)Lnet/minecraft/class_54;
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+		ARG 7 range
 	METHOD method_150 playSound (DDDLjava/lang/String;FF)V
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
+		ARG 7 sound
+	METHOD method_151 getAmbientDarkness (F)I
+		ARG 1 partialTicks
+	METHOD method_152 getSpawnBlockId (II)I
+		ARG 1 x
+		ARG 2 z
+	METHOD method_153 isRegionLoaded (IIII)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 range
+	METHOD method_154 setBlockWithoutNotifyingNeighbors (IIIII)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 blockId
+		ARG 5 meta
 	METHOD method_155 isRegionLoaded (IIIIII)Z
-	METHOD method_156 (IIIIZI)Z
+		ARG 1 minX
+		ARG 2 minY
+		ARG 3 minZ
+		ARG 4 maxX
+		ARG 5 maxY
+		ARG 6 maxZ
+	METHOD method_156 canPlace (IIIIZI)Z
+		ARG 1 blockId
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 fallingBlock
 		ARG 6 side
+	METHOD method_157 setBlockEntity (IIILnet/minecraft/class_55;)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 blockEntity
+	METHOD method_158 getLightLevel (IIIZ)I
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
 	METHOD method_159 setTime (J)V
 		ARG 1 time
+	METHOD method_160 raycast (Lnet/minecraft/class_26;Lnet/minecraft/class_26;)Lnet/minecraft/class_27;
+		ARG 1 start
+		ARG 2 end
+	METHOD method_161 raycast (Lnet/minecraft/class_26;Lnet/minecraft/class_26;Z)Lnet/minecraft/class_27;
+		ARG 1 start
+		ARG 2 end
+	METHOD method_162 raycast (Lnet/minecraft/class_26;Lnet/minecraft/class_26;ZZ)Lnet/minecraft/class_27;
+		ARG 1 start
+		ARG 2 end
+	METHOD method_163 getVisibilityRatio (Lnet/minecraft/class_26;Lnet/minecraft/class_25;)F
+		ARG 1 vec
+		ARG 2 box
+	METHOD method_164 getBrightness (Lnet/minecraft/class_56;III)I
+		ARG 1 type
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_165 updateLight (Lnet/minecraft/class_56;IIII)V
+		ARG 1 lightType
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_166 queueLightUpdate (Lnet/minecraft/class_56;IIIIII)V
+		ARG 1 type
+		ARG 2 minX
+		ARG 3 minY
+		ARG 4 minZ
+		ARG 5 maxX
+		ARG 6 maxY
+		ARG 7 maxZ
+	METHOD method_167 queueLightUpdate (Lnet/minecraft/class_56;IIIIIIZ)V
+		ARG 1 type
+		ARG 2 minX
+		ARG 3 minY
+		ARG 4 minZ
+		ARG 5 maxX
+		ARG 6 maxY
+		ARG 7 maxZ
 	METHOD method_168 canSpawnEntity (Lnet/minecraft/class_25;)Z
 		ARG 1 box
 	METHOD method_169 isBoxInMaterial (Lnet/minecraft/class_25;Lnet/minecraft/class_15;)Z
```

Comment:
Should this be flip flopped to `isMaterialInBox` this is the way it is for MCP `isMaterialInBB`

Reply:
will be `isMaterialInBox`